test/unit/search/fm_index_cursor/fm_index_cursor_collection_test_template.hpp

```diff
@@ -404,7 +404,7 @@ TYPED_TEST_P(fm_index_cursor_collection_test, extend_const_char_pointer)
 
 TYPED_TEST_P(fm_index_cursor_collection_test, concept_check)
 {
-    EXPECT_TRUE(seqan3::fm_index_cursor_specialisation<TypeParam>);
+    EXPECT_TRUE(seqan3::detail::template_specialisation_of<TypeParam, seqan3::fm_index_cursor>);
```

Comment:
```suggestion
    EXPECT_TRUE((seqan3::detail::template_specialisation_of<TypeParam, seqan3::fm_index_cursor>));
```
always wrap in extra parenthesis if there is a comma, otherwise the `EXPECT_[...]`macro doesn't work properly.

Reply:
Thank you!